tools/define-compose-url.sh

```diff
@@ -2,19 +2,20 @@
 set -euo pipefail
 source /etc/os-release
 
-# This isn't needed when not running on RHEL
-if [[ $ID != rhel ]]; then
-  exit 0
+# This isn't needed when not running on RHEL or it's subscribed
+if [[ $ID != rhel ]] || sudo subscription-manager status; then
+  return 0
```

Comment:
This should be `exit 0` instead of `return 0`.

Reply:
`ostree.sh` test is sourcing this file and when this has `exit 0` it will not run the test at all on fedora and centos so I decided to just `return 0` so the test will run as expected... I'm open to suggestions though if you think this is not a good way to resolve this...